internal/cmd/db_create.go

```diff
@@ -307,6 +308,7 @@ func handleInstanceCreationError(client *turso.Client, name, locationId string,
 
 	_, locationId, err := promptSelect.Run()
 	if err != nil {
+		client.Databases.Delete(name)
```

Comment:
When you return the error right after this call, it will trigger a second delete - or won't it?

Reply:
whoops that is true, removed the one inside the function